drawBot/context/tools/drawBotbuiltins.py

```diff
@@ -0,0 +1,26 @@
+
+def norm(value, start, stop):
+    """
+    Return the interpolation factor (between 0 and 1) of a VALUE between START and STOP.
+    """
+    return float(value - start) / float(stop - start)
+
+
+def lerp(start, stop, factor):
+    """
+    Interpolate using a value between 0 and 1
+    """
+    return start + (stop - start) * factor
+
+
+def remap(value, start1, stop1, start2, stop2, clamp=False):
+    """
+    Re-maps a number from one range to another.
+    """
+    factor = norm(value, start1, stop1)
+    if clamp:
+        if factor < 0:
+            factor = 0
+        if factor > 1:
+            factor = 1
```

Comment:
```python
    if clamp:
        factor = min(1, max(factor, 0))
```

Reply:
fine by me, I just find it less readable :)

Reply:
Fine with me, too, but then make the second if an elif.

Reply:
jep, changed it back